packages/x-date-pickers/src/internals/hooks/usePicker/usePickerViews.ts

```diff
@@ -48,6 +69,11 @@ export interface UsePickerViewsBaseProps<TView extends DateOrTimeView> {
    * Array of views to show.
    */
   views: readonly TView[];
+  /**
+   * Define custom view renderers for each section.
+   * If `null`, the view will be editing with the field.
```

Comment:
Is this comment used anywhere?
Each picker type seems to override it and provide it's own comment with one extra line.

Reply:
I don't think it is used anywhere in the JSDoc
It's just here to describe what the param is doing in `usePickerViews`